website/source/partials/builder/vsphere/iso/_CreateConfig-not-required.html.md

```diff
@@ -6,7 +6,7 @@
     the full list of supported VM hardware versions.
     
 -   `guest_os_type` (string) - Set VM OS type. Defaults to `otherGuest`. See [
-    here](https://pubs.vmware.com/vsphere-6-5/index.jsp?topic=%2Fcom.vmware.wssdk.apiref.doc%2Fvim.vm.GuestOsDescriptor.GuestOsIdentifier.html)
+    here](https://code.vmware.com/apis/358/vsphere/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html)
```

Comment:
This documentation is actually auto-generated from the the configuration file https://github.com/hashicorp/packer/blob/eb3561798561a56d31d6634dc65f4f3c222b7a41/builder/vsphere/iso/step_create.go#L34

You can make the fix in that file and run the command `make generate` to update the website docs.